storm-server/src/main/java/org/apache/storm/scheduler/SchedulerAssignmentImpl.java

```diff
@@ -45,7 +43,7 @@
      */
     private final Map<ExecutorDetails, WorkerSlot> executorToSlot = new HashMap<>();
     private final Map<WorkerSlot, WorkerResources> resources = new HashMap<>();
-    private final Map<String, Double> nodeIdToTotalSharedOffHeap = new HashMap<>();
+    private final Map<String, Double> nodeIdToTotalSharedOffHeapNode = new HashMap<>();
```

Comment:
Nit: I'm not sure why adding "Node" to all these variables makes it clearer what they contain. E.g. for this one, I think node id -> shared offheap is more readable than node id -> shared offheap node.

If other people prefer the new names, it's fine though.

Reply:
Trying to make explicit a distinction between sharedOffHeapWorker and sharedOffHeapNode